README.md

```diff
@@ -9,6 +9,9 @@ Discord bot for Eddie Jaoude's Discord server
 - general channel ID of your discord server, read the instructions on one of these links to get yours:
    - [Get the channel ID of the Discord text channel](https://github.com/Chikachi/DiscordIntegration/wiki/How-to-get-a-token-and-channel-ID-for-Discord#get-the-channel-id-of-the-discord-text-channel)
    - [Where can I find my User/Server/Message ID?](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-)
+- ID of your discord server, to get your server ID follow these steps:
+   - First make sure you have **Developer Mode** enabled on your Discord by visiting your Discord settings and going to Appearance
```

Comment:
I could not see how to do this. Can you add a screenshot? (drop the image in an issue / comment and grab the generated URL rather than putting the image in the repo itself